docs/release.md

```diff
@@ -0,0 +1,108 @@
+# Quickstart New Image Playbook
```

Comment:
I struggle with the name here.  

Perhaps, Quickstart Update Guide.

Reply:
How about "Quickstart Release Guide?"

Reply:
sgtm